src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -179,6 +181,13 @@ class ActivityFeed extends React.Component<Props, State> {
         }
     };
 
+    handleToggleHeaderShadow = () => {
+        if (this.feedContainer) {
+            const shouldHaveShadow = this.feedContainer.scrollTop !== 0;
+            this.setState({ isHeaderShadowVisible: shouldHaveShadow });
```

Comment:
minor but how do you feel about using `isScrolled` as the state?

Reply:
Same here.